python/gui/maas_experiment/application_values.py

```diff
@@ -173,6 +173,8 @@ def get_full_localtimezone():
 BASE_DIRECTORY = Path(__file__).resolve().parent.parent
 """The path to the base directory of the service"""
 
+STATIC_ROOT = BASE_DIRECTORY / "static"
```

Comment:
Has this been tested inside and outside of docker? Setting Django's `STATIC` variables can end up having odd or ill or differing effects when placed behind nginx due to routing issues.